packages/clarity-js/src/data/extract.ts

```diff
@@ -1,38 +1,35 @@
 import { ExtractSource, Syntax, Type } from "@clarity-types/core";
 import { Event, Setting, ExtractData } from "@clarity-types/data";
-import config from "@src/core/config";
 import encode from "./encode";
 import * as internal from "@src/diagnostic/internal";
 import { Code, Constant, Severity } from "@clarity-types/data";
 
 export let data: ExtractData = {};
-export let keys: (number | string)[] = [];
-
-let variables : { [key: number]: Syntax[] } = {};
-let selectors : { [key: number]: string } = {};
-export let fragments: string[] = [];
+export let keys: number[] = [];
 
+let variables : { [key: number]: { [key: string]: Syntax[] }} = {};
+let selectors : { [key: number]: { [key: string]: string }} = {};
 export function start(): void {
+    reset();
+}
+
+export function trigger(input: string): void { 
     try {
-        let e = config.extract;
-        if (!e) { return; }
-        for (let i = 0; i < e.length; i+=3) {
-            let source = e[i] as ExtractSource;
-            let key = e[i+1] as number;
+        var parts = input && input.length > 0 ? input.split(/ (.*)/) : [Constant.Empty];
```

Comment:
Wasn't this already split on earlier in the action handler parsing routine in upload.ts?

Reply:
this regex splits by the first space - so the parts are different.